src/SFA.DAS.AdminService.Web/Views/Application/WithdrawalApplication/_WithdrawalApplicationsPartial.cshtml

```diff
@@ -21,28 +21,26 @@
     <thead class="govuk-table__head">
         <tr class="govuk-table__row">
             <th scope="col" class="govuk-table__header govuk-!-width-one-quarter">
-                <a class="govuk-link das-no-wrap">Type</a>
-            </th>
-            <th scope="col" class="govuk-table__header govuk-!-width-two-thirds">
                 <sfa-sortable-column-header class=@(Model.Fragment + "-sort govuk-link" + " das-no-wrap")
                                             data-sortColumn="@WithdrawalApplicationsSortColumn.Name"
                                             asp-action="@Model.SortColumnAction"
                                             asp-controller="WithdrawalApplication"
                                             asp-fragment="@Model.Fragment"
                                             sfa-sort-column="@WithdrawalApplicationsSortColumn.Name"
                                             sfa-table-sort-column="@Model.SortColumn"
-                                            sfa-sort-direction="@Model.SortDirection">Name</sfa-sortable-column-header>
+                                            sfa-sort-direction="@Model.SortDirection">Organisation</sfa-sortable-column-header>
             </th>
-            <th scope="col" class="govuk-table__header govuk-!-width-one-quarter govuk-table__header--numeric">
+            <th scope="col" class="govuk-table__header govuk-!-width-two-thirds">
                 <sfa-sortable-column-header class=@(Model.Fragment + "-sort govuk-link" + " das-no-wrap")
-                                            data-sortColumn="@WithdrawalApplicationsSortColumn.GovernanceRecommendation"
+                                            data-sortColumn="@WithdrawalApplicationsSortColumn.Type"
```

Comment:
Why is this not the name of the column - 'Details'

Reply:
I've renamed the column sort type